spec/lib/attache/upload_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:params) { {} }
   let(:filename) { "Exãmple %#{rand} %20.gif" }
   let(:file) { StringIO.new(IO.binread("spec/fixtures/landscape.jpg"), 'rb') }
+  let(:base64_data) { "data:image/jpeg;base64,/9j/4QBiRXhpZgAATU0AKgAAAAgABQESAAMAAAABAAUAAAEaAAUAAAABAAAASgEbAAUAAAABAAAAUgEoAAMAAAABAAIAAAITAAMAAAABAAEAAAAAAAAAAABIAAAAAQAAAEgAAAAB/9sAQwACAgICAgECAgICAwICAwMGBAMDAwMHBQUEBggHCQgIBwgICQoNCwkKDAoICAsPCwwNDg4PDgkLEBEQDhENDg4O/9sAQwECAwMDAwMHBAQHDgkICQ4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4ODg4O/8AAEQgABAADAwEiAAIRAQMRAf/EABUAAQEAAAAAAAAAAAAAAAAAAAAJ/8QAHxAAAQMEAwEAAAAAAAAAAAAAAQIDBAAFBhEHEjEh/8QAFQEBAQAAAAAAAAAAAAAAAAAABAX/xAAaEQACAgMAAAAAAAAAAAAAAAABAgARMdHh/9oADAMBAAIRAxEAPwCZefZjIj815UzGsNgjR0XN4NsosrJS2nudJGwToeD75SlKjI7FBzUSC1Zn/9k=" }
```

Comment:
What was the string before encoded?

I suggest change to

`Base64.encode64("original string")`

easier to understand what we are testing :smile: